### PR TITLE
Allows you to put food on the IV drips again.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

See title. Thanks to how the edible component works and some clever usage of byond, this works just fine!

## Why It's Good For The Game

Steak IVs are the funniest shit I ever seen.

## Changelog
:cl:
fix: You can now put food on the IV drip again.
/:cl:

### CODE REVIEW

File: code/game/machinery/iv_drip.dm [hunk 1]
```diff
@@ -100,7 +100,7 @@
 /obj/machinery/iv_drip/attackby(obj/item/W, mob/user, params)
 	if(use_internal_storage)
 		return ..()
-
+
```
Comment (Qustinnus) [inline code/game/machinery/iv_drip.dm:103]: yeah looks good to me

**Comment by Mothblocks** [comment]
image

great code sir

File: code/game/machinery/iv_drip.dm [hunk 2]
```diff
@@ -16,7 +16,7 @@
 	///whether we feed slower
 	var/dripfeed = FALSE
 	///Internal beaker
-	var/obj/item/reagent_containers/beaker
+	var/obj/beaker
```
Comment (Qustinnus) [inline code/game/machinery/iv_drip.dm:19]: rename this as its not a beaker

**Comment by ATH1909** [comment]
that steak got drip ⭐ 💯 😮

**Pull request merged**